scripts/build-ios-framework.sh

```diff
@@ -0,0 +1,47 @@
+cd rust-core
```

Comment:
```suggestion
THIS_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
cd $THIS_DIR/../rust-core
```

itchy bash neckbeard that allows us to run this script from any directory bc it `cd`s _from_ the `scripts` dir. clearly paramount to the success of this exploration